assets/less/cds-rdm/globals/site.variables

```diff
@@ -48,10 +48,11 @@
 @placeholderTextColor: #abadb1;
 
 @recordsListTitleBackground: #313741;
-@recordsListMetadataBackground: #d6dde8;
+@metadataBackground: #d6dde8;
```

Comment:
why not to use the previous variable name ? did something change in the base theme ?

Reply:
Previously this background color was only used by the recordsList whilst now, it isn't.